code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -330,7 +330,7 @@
 			radiation_pulse(location,rad_power)
 
 		var/new_heat_capacity = air.heat_capacity()
-		if(new_heat_capacity > MINIMUM_HEAT_CAPACITY && (air.temperature <= FUSION_MAXIMUM_TEMPERATURE || reaction_energy <= 0))	//If above FUSION_MAXIMUM_TEMPERATURE, will only adjust temperature for endothermic reactions.
+		if(new_heat_capacity > MINIMUM_HEAT_CAPACITY && (air.temperature <= (FUSION_MAXIMUM_TEMPERATURE*(CONFIG_GET(number/fusioncap))) || reaction_energy <= 0))	//If above FUSION_MAXIMUM_TEMPERATURE, will only adjust temperature for endothermic reactions. Austation -- The fusion temperature limit can be modified in server config
```

Comment:
```air.temperature <= (FUSION_MAXIMUM_TEMPERATURE*(CONFIG_GET(number/fusioncap)))```
the funny thing is, if the config value is zero, the max temp * the config is also zero, which means the cap will always trigger lmfao. Please fix this.

Reply:
It's hacky, but it _does_ work.

Reply:
Why do I review things at midnight.
Yeah I misunderstood this. Add a changelog entry to changelog.txt and we're good to go